test/test_ops.py

```diff
@@ -110,6 +110,8 @@ def __new__(cls, name, variant_test_name=""):
             AllowedOpInfoEntry('linalg.cholesky'),
             AllowedOpInfoEntry('linalg.cholesky_ex'),
             AllowedOpInfoEntry('linalg.eig'),
+            # Tests fail because absolute values of eigenvectors should be compared
+            # AllowedOpInfoEntry('linalg.eigh'),
```

Comment:
nit, can we move this line back to the below disabled list?

Reply:
Done!